src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt

```diff
@@ -0,0 +1,108 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.resource
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.springframework.http.HttpStatus
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.manageusersapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.manageusersapi.service.UserExistsException
+import uk.gov.justice.digital.hmpps.manageusersapi.service.UserService
+import javax.validation.Valid
+import javax.validation.constraints.Email
+import javax.validation.constraints.NotBlank
+
+@RestController
+@Validated
+class UserController(
+  private val userService: UserService
+) {
+
+  @PostMapping("/users")
+  @Throws(UserExistsException::class)
+  @PreAuthorize("hasRole('ROLE_CREATE_USER')")
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Create a DPS user",
+    description = "Creates a specific DPS user. Requires role ROLE_CREATE_USER",
+    security = [SecurityRequirement(name = "ROLE_CREATE_USER")],
+    responses = [
+      ApiResponse(
+        responseCode = "200",
```

Comment:
Do we need to add the 200 response or are they generated automatically if not there